pages.hi/common/cat.md

```diff
@@ -0,0 +1,23 @@
+# cat
+
+> फ़ाइलों को प्रिंट और संक्षिप्त करें।
+
+- मानक आउटपुट में फ़ाइल की सामग्री प्रिंट करें:
+
+`cat {{फ़ाइल}}`
+
+- लक्ष्य फ़ाइल में कई फ़ाइलों को संयुक्त करें:
+
+`cat {{फ़ाइल1}} {{फ़ाइल2}} > {{लक्ष्य_फ़ाइल}}`
+
+- लक्ष्य फ़ाइल में कई फ़ाइलें जोड़ें:
```

Comment:
The translation for `Append several files into the target file` is not the same. The files' content are added to the end of the `target_file`.
![Screenshot_20201009_141922](https://user-images.githubusercontent.com/1536740/95612885-71e2c500-0a3a-11eb-96db-400505e9d050.png)

Reply:
Just 1 pending comment @Athi223. Thanks!

Reply:
Updated. to be honest there's no exact word that means 'append' so I just tried to get it as close as possible.